lib/diaspora/commentable.rb

```diff
@@ -13,12 +13,12 @@ def self.included(model)
     end
 
     # @return [Array<Comment>]
-  def last_three_comments
-    return [] if self.comments_count == 0
-    # DO NOT USE .last(3) HERE.  IT WILL FETCH ALL COMMENTS AND RETURN THE LAST THREE
-    # INSTEAD OF DOING THE FOLLOWING, AS EXPECTED (THX AR):
-    self.comments.order('created_at DESC').limit(3).includes(:author => :profile).reverse
-  end
+    def last_three_comments
+      return [] if self.comments_count == 0
+      # DO NOT USE .last(3) HERE.  IT WILL FETCH ALL COMMENTS AND RETURN THE LAST THREE
+      # INSTEAD OF DOING THE FOLLOWING, AS EXPECTED (THX AR):
+      self.comments.order('created_at DESC').limit(3).includes(:author => :profile).reverse
```

Comment:
- Style/RedundantSelf: Redundant `self` detected.
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.